### PR TITLE
bears/__init__.py: import constants

### PR DESCRIPTION
**Pull request by Nosferatul**
Add the version constants from Constants.py. Add variable "__version__".

Fixes https://github.com/coala-analyzer/coala-bears/issues/84

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
Maybe you could add a space in the commit title after the ':' ?

**Comment by Adrianzatreanu** [comment]
I meant the commit title. not the PR title. you can do this using git commit --amend and pushing it again

File: bears/__init__.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import sys
+from bears import Constants.py
```
Comment (SanketDG) [inline bears/__init__.py]: @Nosferatul do you think this will work?

**Comment by SanketDG** [comment]
unack 8b3c02b

**Comment by Adrianzatreanu** [comment]
why is gitmate taking so long? any ideas?

**Comment by AbdealiJK** [comment]
@Adrianzatreanu gitmate is beta and seems to be broken (it hasnt been working for 2-3 days i think)

**Comment by AbdealiJK** [comment]
ack f73ad6f

**Comment by AbdealiJK** [comment]
@rultor merge

**Comment by AbdealiJK** [comment]
@Nosferatul Awesome work !
You got your first commit into coala :)

**Pull request merged**